### ISSUE TITLE
Cannot create an API in APIM with Analytics

### ISSUE THREAD

**Issue by ThilinaManamgoda** [wso2/docker-apim#146]
**Description:**
Cannot create an API in APMI with Analytics docker-compose deployment. Throws the following error,
`com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Table 'apimgt_db.AM_LABELS' doesn't exist
api-manager_1   | 	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
api-manager_1   | 	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
api-manager_1   | 	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
api-manager_1   | 	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
api-manager_1   | 	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
api-manager_1   | 	at com.mysql.jdbc.Util.getInstance(Util.java:408)
api-manager_1   | 	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:944)
api-manager_1   | 	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3973)
api-manager_1   | 	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3909)
api-manager_1   | 	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2527)
api-manager_1   | 	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2680)
api-manager_1   | 	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2484)
api-manager_1   | 	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1858)
api-manager_1   | 	at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:1966)
api-manager_1   | 	at sun.reflect.GeneratedMethodAccessor36.invoke(Unknown Source)
api-manager_1   | 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
api-manager_1   | 	at java.lang.reflect.Method.invoke(Method.java:498)
api-manager_1   | 	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
api-manager_1   | 	at com.sun.proxy.$Proxy19.executeQuery(Unknown Source)
api-manager_1   | 	at org.wso2.carbon.apimgt.impl.dao.ApiMgtDAO.getAllLabels(ApiMgtDAO.java:11527)
api-manager_1   | 	at org.wso2.carbon.apimgt.impl.utils.APIUtil.getAllLabels(APIUtil.java:6631)
api-manager_1   | 	at org.wso2.carbon.apimgt.impl.utils.APIUtil.attachLabelsToAPIArtifact(APIUtil.java:1009)
api-manager_1   | 	at org.wso2.carbon.apimgt.impl.utils.APIUtil.createAPIArtifactContent(APIUtil.java:987)
api-manager_1   | 	at org.wso2.carbon.apimgt.impl.APIProviderImpl.createAPI(APIProviderImpl.java:2599)
api-manager_1   | 	at org.wso2.carbon.apimgt.impl.APIProviderImpl.addAPI(APIProviderImpl.java:672)
api-manager_1   | 	at org.wso2.carbon.apimgt.impl.UserAwareAPIProvider.addAPI(UserAwareAPIProvider.java:59)
api-manager_1   | 	at org.wso2.carbon.apimgt.hostobjects.APIProviderHostObject.saveAPI(APIProviderHostObject.java:1158)
api-manager_1   | 	at org.wso2.carbon.apimgt.hostobjects.APIProviderHostObject.jsFunction_createAPI(APIProviderHostObject.java:1058)
api-manager_1   | 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
api-manager_1   | 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
api-manager_1   | 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
api-manager_1   | 	at java.lang.reflect.Method.invoke(Method.java:498)
api-manager_1   | 	at org.mozilla.javascript.MemberBox.invoke(MemberBox.java:126)
api-manager_1   | 	at org.mozilla.javascript.FunctionObject.call(FunctionObject.java:386)
api-manager_1   | 	at org.mozilla.javascript.optimizer.OptRuntime.call1(OptRuntime.java:32)
api-manager_1   | 	at org.jaggeryjs.rhino.publisher.modules.api.c3._c_anonymous_5(/publisher/modules/api/add.jag:108)
api-manager_1   | 	at org.jaggeryjs.rhino.publisher.modules.api.c3.call(/publisher/modules/api/add.jag)
api-manager_1   | 	at org.mozilla.javascript.ScriptRuntime.applyOrCall(ScriptRuntime.java:2430)
api-manager_1   | 	at org.mozilla.javascript.BaseFunction.execIdCall(BaseFunction.java:269)
api-manager_1   | 	at org.mozilla.javascript.IdFunctionObject.call(IdFunctionObject.java:97)
api-manager_1   | 	at org.mozilla.javascript.optimizer.OptRuntime.call2(OptRuntime.java:42)
api-manager_1   | 	at org.jaggeryjs.rhino.publisher.modules.api.c0._c_anonymous_42(/publisher/modules/api/module.jag:133)
api-manager_1   | 	at org.jaggeryjs.rhino.publisher.modules.api.c0.call(/publisher/modules/api/module.jag)
api-manager_1   | 	at org.mozilla.javascript.optimizer.OptRuntime.call1(OptRuntime.java:32)
api-manager_1   | 	at org.jaggeryjs.rhino.publisher.site.blocks.item_design.ajax.c0._c_anonymous_5(/publisher/site/blocks/item-design/ajax/add.jag:494)
`

**Suggested Labels:**
Type/Bug

**Affected Product Version:**
WSO2 APIM 2.5.0

**Steps to reproduce:**
Try to add a REST endpoint in the API definition section in Design API flow in the Publisher.

**Comment by chirangaalwis** [comment]
@ThilinaManamgoda the above mentioned issue IMK has been fixed. Please check https://github.com/wso2/docker-apim/blob/2.5.x/docker-compose/apim-with-analytics/mysql/scripts/apim_mysql5.7.sql#L1405.

The issue was fixed in https://github.com/wso2/docker-apim/pull/135.
